src/style.css

```diff
@@ -1,3 +1,67 @@
+nav,
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
Setting `box-sizing` individually for each class is not a great idea - as there are few scenarios when one needs to vary it.

In most cases, including yours, it is better to set a single box-sizing value for all elements and vary it only when needed

Try to fix this issue - it looks minor in this task but may become painful in large projects.

Here is a good article on box-sizing: https://css-tricks.com/box-sizing/

And here is a good way to set box-sizing: https://css-tricks.com/box-sizing/#aa-good-better-and-probably-best-box-sizing-reset-methods